Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -0,0 +1,148 @@
+---
+versionFrom: 8.7.0
+---
+
+# Block List
+
+`Alias: Umbraco.BlockList`
+
+`Returns: TODO: Jet-To-Be-Written-`
+
+
+**Block List** is a list editing property editor, using Element Types to define the list item schema.
+
+
+## Configure Block List
+
+The **Block List** property editor is configured in the same way as any standard property editor, via the *Data Types* admin interface. To set up your Block List Editor property, create a new *Data Type* and select **Block List** from the list of available property editors.
+
+Then you will see the configuration options for a **Block List** as shown below.
+
+![Block List - Data Type Definition](images/BlockListEditor_DataType.png)
+
+The Data Type editor allows you to configure the following properties:
+
+- **Available Blocks** - Here you will define the Block Types to be available for use in the propety. Read more on how to set up Block Types below.
+- **Amount** - Sets the minimum and/or maximum number of items that should be allowed in the list.
+- **Inline editing mode** - Enabling this will change editing experience to inline, meaning that editing the data of blocks happens item deletions to require a confirmation before being deleted. Defaults to `true`.
```

Comment:
I'm not entirely sure I understand this line:
"... meaning that editing the data of blocks happens item deletions to require a confirmation before being deleted"

Might be a typo somewhere, but I cannot figure out what that sentence is supposed to mean? 🤔
Can you help me out?

Reply:
I've changed this, could you check it again?